app/_locales/en/messages.json

```
@@ -86,7 +86,7 @@
     "message": "Activity"
   },
   "activityLog": {
-    "message": "activity log"
+    "message": "Activity Log"
```

Generally designs lately have capitalized just the first letter of each title, rather than every word. That is probably what should be done here as well

```suggestion
    "message": "Activity log"
```